bundle/config/mutator/rewrite_workspace_prefix.go

```diff
@@ -44,7 +44,7 @@ func (m *rewriteWorkspacePrefix) Apply(ctx context.Context, b *bundle.Bundle) di
 				if strings.Contains(vv, path) {
 					diags = append(diags, diag.Diagnostic{
 						Severity:  diag.Warning,
-						Summary:   fmt.Sprintf("substring %q found in %q. Please update this to %q. For more information, please refer to: https://github.com/databricks/cli/blob/main/docs/release_notes/workspace_prefix.md", path, vv, strings.Replace(vv, "/Workspace/", "", 1)),
+						Summary:   fmt.Sprintf("substring %q found in %q. Please update this to %q. For more information, please refer to: https://docs.databricks.com/en/release-notes/bundles.html#workspace-paths-will-be-automatically-prefixed", path, vv, strings.Replace(vv, "/Workspace/", "", 1)),
```

Comment:
```suggestion
						Summary:   fmt.Sprintf("substring %q found in %q. Please update this to %q. For more information, please refer to: https://docs.databricks.com/en/release-notes/bundles.html#workspace-paths", path, vv, strings.Replace(vv, "/Workspace/", "", 1)),
```
I put added a simpler anchor in that article heading.

Reply:
I also would take out 'please': "... Update this to %q. For more information, see ..."